builder/image/templates/builder

```diff
@@ -101,7 +101,7 @@ if [ ! -f Dockerfile ]; then
     if [ -f /buildpacks ]; then
         BUILD_OPTS+=' -v /buildpacks:/tmp/buildpacks:rw'
         # give non-root slugbuilder user R/W perms for docker volumes
-        chown -R 2000:2000 /buildpacks
+        sudo chown -R 2000:2000 /buildpacks
```

Comment:
The docker daemon is running as root (thats why sudo was required) and all commands before https://github.com/deis/deis/blob/master/builder/image/slugbuilder/Dockerfile#L8 are executed as root, so chown-ing the buildpacks dir before switching the user should work (if it was mounted).